src/fake_node_modules/powercord/components/ContextMenu/Button.jsx

```diff
@@ -43,4 +48,16 @@ module.exports = class ButtonItem extends React.PureComponent {
 
     return button;
   }
+
+  getButtonImage () {
+    return (
+      this.props.image.startsWith('fa-')
+        ? <div style={{ cursor: 'pointer!important' }}
+          class={`${this.props.image.endsWith('-alt')
+            ? 'far'
+            : 'fas'}
+            ${this.props.image.replace('-alt', '')} fa-fw`} />
+        : <img src={this.props.image} />
```

Comment:
alt attribute is required on img tags in HTML specifications. Empty alt will do the trick